CodeEdit/Features/Settings/Models/SettingsData.swift

```diff
@@ -61,7 +62,42 @@ struct SettingsData: Codable, Hashable {
             SourceControlSettings.self,
             forKey: .sourceControl
         ) ?? .init()
-        self.keybindings = try container.decodeIfPresent(KeybindingsSettings.self, forKey: .keybindings) ?? .init()
+        self.keybindings = try container.decodeIfPresent(
+            KeybindingsSettings.self,
+            forKey: .keybindings
+        ) ?? .init()
         self.featureFlags = try container.decodeIfPresent(FeatureFlagsSettings.self, forKey: .featureFlags) ?? .init()
     }
+
+    // swiftlint:disable cyclomatic_complexity
```

Comment:
Cyclomatic complexity should be fixed. Is there no better way to this?

Reply:
sorry, there isn't :c